modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -0,0 +1,62 @@
+= Redpanda Cloud API Overview
+:description: Learn about using the Cloud API to manage clusters and cloud resources.
+:page-cloud: true
+:page-api: cloud
+:page-api-slot: overview
+:page-beta: true
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
+
+== Requirements
+
+To be able to use the Cloud API:
+
+* You must be a customer with an existing organization in Redpanda Cloud.
+* When you authenticate, the access token granted to you is associated with a specific Redpanda Cloud organization. If you want to use the API for a different organization, you must acquire a new token through a service account with that organization. See xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Cloud API Authentication] for steps to authenticate API requests.
+
+== Cloud API architecture
+
+Redpanda Cloud uses a control plane and data plane architecture. The *control plane* is where most cluster management, operations, and maintenance takes place. The control plane enforces rules in the data plane. The *data plane* is where your cloud cluster lives. The term _data plane_ is used interchangeably with _cluster_. For BYOC customers, the data plane is in your VPC. To learn more about the Redpanda Cloud platform, see the xref:deploy:deployment-option/cloud/cloud-overview.adoc[overview].
+
+The *Control Plane API* allows you to interact with the control plane. You use the Control Plane API to manage cloud clusters, organization, resource groups, networks, and other cloud resources. The Control Plane API does not interact directly with Redpanda clusters, although some of the actions performed through the API can eventually make their way to a cluster. See xref:deploy:deployment-option/cloud/api/cloud-controlplane-api.adoc[Manage Clusters using the Cloud API] for steps to create and delete clusters, or see the full xref:api:ROOT:cloud-api.adoc[Cloud API Reference documentation]. 
+
+The *Data Plane APIs* are a collection of APIs through which you can perform actions directly against a Redpanda cluster. These APIs expose more high-level operations on the data plane by acting as a wrapper around the other APIs supported by Redpanda (Kafka API, Admin API, or Schema Registry API). See xref:deploy:deployment-option/cloud/api/cloud-dataplane-api.adoc[Manage Cluster Resources Using the Cloud API] for steps to create an access-control list (ACL), topic, or connector, or see the full xref:api:ROOT:cloud-api.adoc[Cloud API Reference documentation].
+
+NOTE: For more information on the APIs available with Redpanda, see the xref:reference:api-reference.adoc[API and SDK Reference documentation].
```

Comment:
```suggestion
See also:

* xref:deploy:deployment-option/cloud/api/cloud-dataplane-api.adoc[Manage Cluster Resources Using the Cloud API] for steps to create an access-control list (ACL), topic, or connector
* xref:api:ROOT:cloud-api.adoc[Cloud API docs] or the complete xref:reference:api-reference.adoc[API and SDK Reference documentation] 
```
Suggest putting all 3 links together as a list instead of inline + note.